src/geouned/GEOUNED/Utils/Geometry_GU.py

```diff
@@ -136,9 +136,9 @@ def __same_torus_surf__(self, torusList):
                 if is_same_torus(
```

Comment:
to be consistent with ```is_same_plane``` and ```is_same_cylinder``` functions in ```is_same_torus``` and other "is_same_surface" functions we shoud pass the ```tolerances``` class instead of their attributes.
We pass the ```tolerances``` class to ```is_same_plane``` and ```is_same_cylinder``` functions because we need tolerances for ```Fuzzy``` function, but if in the future we extend Fuzzy functions to other surfaces we will need also the  ```tolerances``` class.